### PR TITLE
[IOTDB-2179] New IT framework supports Windows platform

### PR DESCRIPTION
**Pull request by ericpai**
I have tested this in my PC with WIndows 10 installed.
Hope this PR can help the Windows based developers :)

### CODE REVIEW

**Comment by ericpai** [comment]
@irvine0109 PTAL

**Comment by irvine0109** [comment]
Well done!

Maybe we need to change the .github/workflows/cluster.yml to run cluster-IT in windows at last.

File: integration/src/main/java/org/apache/iotdb/integration/env/ClusterEnvBase.java [hunk 1]
```diff
@@ -124,17 +123,8 @@ public void testWorking() throws InterruptedException {
...
-            "CREATE TIMESERIES root.test" + counter + ".d0.s0 WITH DATATYPE=INT32, ENCODING=RLE");
-        if (statement.execute("SHOW TIMESERIES")) {
-          ResultSet resultSet = statement.getResultSet();
-          if (resultSet.next()) {
-            statement.execute("DELETE STORAGE GROUP root.*");
-            break;
-          }
-        }
-
+        statement.execute("SHOW TIMESERIES");
```
Comment (irvine0109) [inline integration/src/main/java/org/apache/iotdb/integration/env/ClusterEnvBase.java:126]: The statement "SHOW TIMESERIES" may return an empty result when the cluster is not already.

Comment (ericpai) [reply integration/src/main/java/org/apache/iotdb/integration/env/ClusterEnvBase.java:126]: In this PR https://github.com/apache/iotdb/pull/4388 , any error responsed by other nodes will be handled and returned client with a MetaDataException.

**Comment by HTHou** [comment]
Any update of this PR?

**Comment by ericpai** [comment]
> Any update of this PR?

Currently in Windows platform, the failure cases caused by cluster being not ready is often, I'm investigating the issue now. This PR will be temporarily switched to a draft one.

**Comment by ericpai** [comment]
This work will be migrated to new cluster architecture, so this PR is closed. We can move to another one.

**Pull request closed without merging**